pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -337,7 +338,8 @@ public ServerType getServerType() {
        * During realtime segment completion, the value of this enum decides how  non-winner servers should replace  the completed segment.
        */
       public enum CompletionMode {
-        DEFAULT, // default behavior - if the in memory segment in the non-winner server is equivalent to the committed segment, then build and replace, else download
+        DEFAULT,
```

Comment:
auto formatting

Reply:
Can you move the comments for DEFAULT before the value for readability?